vsphere/tests/fixtures/metrics_historical_values_no_dsc_folder.json

```diff
@@ -0,0 +1,1699 @@
+[
```

Comment:
This file is an exact copy of `metrics_historical_values.json` but without the `vsphere_datastore_cluster` tag, correct?

Could we instead generate this file dynamically from the other? Otherwise we'd have to maintain those two json files if we introduce a behavioral change to the check.

Reply:
Correct. 
Yes, it would definitely be better not to duplicate the test file. Let me review this code.

Thanks for the report!